lib/sneakers/workergroup.rb

```diff
@@ -52,5 +54,13 @@ def stop
       @stop_flag.set!
     end
 
+    def create_bunny_connection
+      Bunny.new(Sneakers::CONFIG[:amqp], :vhost => Sneakers::CONFIG[:vhost], :heartbeat => Sneakers::CONFIG[:heartbeat], :logger => Sneakers::logger)
+    end
+
+    def create_connection_or_nil
```

Comment:
sure. no problem. sorry for the delay. i commented out the invocation of the function. hope it's rather helpfull